wordssite/words/models.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+import uuid
+from django.db import models
+from django.utils import timezone
+from django.conf.global_settings import LANGUAGES
+
+
+class BaseModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+    
+class UUIDBaseModel(models.Model):
```

Comment:
if we're not inheriting from BaseModel, then we don't need it. But maybe we could?

Reply:
That's true because we only have Word and Translation models and they are both using  UUIDBaseModel